src/organisms/documentation/search/results.js

```diff
@@ -0,0 +1,93 @@
+import React, { useMemo } from "react"
+import styled from "styled-components"
+import { Tabs, Tab } from "src/components/tabs"
+import Flex from "src/components/templates/flex"
+import { Text } from "src/components/typography"
+
+const Link = styled.a`
+  text-decoration: none;
+`
+
+const Section = ({ title, description, url }) => (
+  <Flex gap={2} column>
+    <Text strong>{title?.snippet || "Untitled"}</Text>
+    {!!description && <Text>{description?.snippet || "No description"}&hellip;</Text>}
+    <Flex alignSelf="end">
+      <Text as={Link} color="primary" href={url} target="_blank">
```

Comment:
If the `url={url?.raw}` can be falsy, we could handle it without providing an invalid link

Reply:
it can't be I will remove the `?`